bucket/komorebi.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.1.7",
+    "description": "A tiling window manager for Windows",
+    "homepage": "https://github.com/LGUG2Z/komorebi",
+    "license": "MIT",
+    "notes": [
+        "Run 'cp $original_dir\\komorebi.sample.ahk $env:USERPROFILE\\komorebi.ahk' to get started with the sample configuration.",
```

Comment:
```suggestion
        "Run 'copy $original_dir\\komorebi.sample.ahk $env:USERPROFILE\\komorebi.ahk' to get started with the sample configuration.",
```

Reply:
Putting in `Copy-Item` for completeness.